packages/components/empty-state/src/MissingContent/MissingContent.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'MissingContent'
+type: 'component'
+slug: /empty-state/missingcontent/
+section: 'utils'
+typescript: ./MissingContent.tsx
+---
```

Comment:
should this be on alpha? Or if it's already as stable should it be under f36-components and sync the versions?

Reply:
as this is so tiny, i think an alpha is maybe a little overkill?